tests/tm_csvs/make_lamb_stew.csv

```diff
@@ -1,56 +0,0 @@
-STEP 1,Unnamed: 1
-input-objects,
-lamb.n.01_1,
-output-objects,
-diced__lamb.n.01_1,
```

Comment:
why do we remove these csv files? Cuz the task no longer exist?

Reply:
Either it doesn't exist or it's no longer a TM recipe